data/migrations/20181011141014_instructions_table.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('instructions', (tbl) => {
+        tbl.increments();
+        tbl.integer('recipe').unsigned().references('id').inTable('recipes');
+        tbl.string('step').notNullable();
```

Comment:
It would be better to create a table of `steps` instead of making a string of multiple steps. The `steps` table would have a many-to-many relationship with the `recipes` table.